{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "a6f5f2c7_02444a1b",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 22873
      },
      "writtenOn": "2023-10-05T12:16:57Z",
      "side": 1,
      "message": "lgtm",
      "revId": "e11905a6470df34637fa71a51e6ed446aab7df50",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "2b34fd67_e156bd77",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 32240
      },
      "writtenOn": "2023-10-05T13:57:48Z",
      "side": 1,
      "message": "recheck - tempest failures\n```\n    tempest.lib.exceptions.IdentityError: Got identity error\nDetails: Unexpected status code 500\n```",
      "revId": "e11905a6470df34637fa71a51e6ed446aab7df50",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}